targets/libjpeg-turbo/libjpeg_turbo_fuzzer.cc

```diff
@@ -0,0 +1,28 @@
+#include <stdint.h>
+#include <stdlib.h>
+
+#include <turbojpeg.h>
+
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
+    tjhandle jpegDecompressor = tjInitDecompress();
+
+    int width, height, subsamp, colorspace;
+    int res = tjDecompressHeader3(
+        jpegDecompressor, data, size, &width, &height, &subsamp, &colorspace);
+
+    if (res != 0 || width == 0 || height == 0) {
+        tjDestroy(jpegDecompressor);
+        return 0;
+    }
+
+    // TODO: this can't possibly be right?
+    void *buf = malloc(width * height * 3);
```

Comment:
since this is .cc file, new & delete please. The cast below won't be necessary too.